From bdf6d5faff979c130f89c2cd6e82553bcc967b60 Mon Sep 17 00:00:00 2001
From: Yankun Zheng <zyk@rock-chips.com>
Date: Tue, 12 Dec 2017 16:06:25 +0800
Subject: [PATCH] ARM: dts: rk3229-at-gva: delete mode-bootloader property

The reboot bootloader flag has been written in misc,
do not need to write in the register

Change-Id: I161b94d554c3a0cb21f6d85b981a247aa1b110ff
Signed-off-by: Yankun Zheng <zyk@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-at-gva.dts | 4 ++++
 arch/arm/boot/dts/rk322x.dtsi       | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3229-at-gva.dts b/arch/arm/boot/dts/rk3229-at-gva.dts
index 12ca32f6e438..2cb75a4b2ef5 100644
--- a/arch/arm/boot/dts/rk3229-at-gva.dts
+++ b/arch/arm/boot/dts/rk3229-at-gva.dts
@@ -147,6 +147,10 @@
 	};
 };
 
+&reboot_mode {
+	/delete-property/ mode-bootloader;
+};
+
 &sdio {
 	max-frequency = <150000000>;
 	mmc-pwrseq = <&sdio_pwrseq>;
diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index 2f6912a727ed..a794b88653c2 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -240,7 +240,7 @@
 			status = "disabled";
 		};
 
-		reboot-mode {
+		reboot_mode: reboot-mode {
 			compatible = "syscon-reboot-mode";
 			offset = <0x5c8>;
 			mode-normal = <BOOT_NORMAL>;
-- 
2.35.3

